server/views/pages/calendar.njk

```diff
@@ -53,6 +54,13 @@
       </div>
     {% endif %}
 
+    {{ govukBreadcrumbs({
+      collapseOnMobile: false,
```

Comment:
shouldn't this be `true`?  On mobile users won't be able to access DPS.

Reply:
according to gov-uk docs, true just means it only shows the first and last items on mobile devices, so not applicable here